application/controllers/Users.php

```diff
@@ -0,0 +1,95 @@
+<?php
```

Comment:
Please be aware of the indentation, make sure all the indentation are consistent
Ex: 1 tab = 4 spaces